exa/core/box.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2015-2020, Exa Analytics Development Team
+# Distributed under the terms of the Apache License 2.0
+"""
+Data container
+####################################
+An in-memory related data container.
+"""
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import networkx as nx
+from traitlets import Unicode, Dict
+
+import exa
+from exa import Base
+from exa.core.data import Data
+
+class Box(Base):
+    name = Unicode(help="string name of container")
+    meta = Dict(help="metadata dictionary")
+    description = Unicode(help="container description")
+
+    def copy(self, **kws):
+        return self.__class__(**kws)
```

Comment:
I suppose this will need to iterate over all data objects, copy them, and pass them in as kw/args to the constructor?